### PR TITLE
feat(dtfs2-7069): update dtfs helpdesk hyperlink

### PR DESCRIPTION
**Pull request by AlexBramhill**
## Introduction :pencil2:
update email to use new helpdesk hyperlink, update accessibility to use env var for email address

## Resolution :heavy_check_mark:
use env vars to determine whether to display helpdesk hyperlink

### CODE REVIEW

File: docker-compose.gha.yml [hunk 1]
```diff
@@ -83,6 +83,8 @@ services:
       - NODE_ENV=${NODE_ENV}
       - RATE_LIMIT_THRESHOLD
       - UTILISATION_REPORT_DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH
+      - CONTACT_US_SELF_SERVICE_PORTAL_URL
```
Comment (abhi-markan) [inline docker-compose.gha.yml:86]: Does CONTACT_US_EMAIL_ADDRESS needs to be added to 'portal' on all the docker compose please?

File: trade-finance-manager-ui/templates/accessibility-statement.njk [hunk 2]
```diff
@@ -37,12 +37,12 @@
...
   <ul class="govuk-list accessibility-statement-list">
-    <li>DigitalService.TradeFinance@ukexportfinance.gov.uk </li>
+    <li>{{ CONTACT_US_EMAIL_ADDRESS }}</li>
   </ul>
   <p class="govuk-body">We'll consider your request and get back to you in 2 days.</p>
 
   <h2 class="govuk-heading-m" data-cy="reporting-accessibility-problems-heading">Reporting accessibility problems with this website</h2>
-  <p class="govuk-body" data-cy="contact-dtfs">We're always looking to improve the accessibility of this website. If you find any additional problems not listed above or think we're not meeting accessibility requirements, contact: <a href="mailTo:DigitalService.TradeFinance@ukexportfinance.gov.uk">DigitalService.TradeFinance@ukexportfinance.gov.uk</a>.</p>
+    <p class="govuk-body" data-cy="contact-dtfs">We're always looking to improve the accessibility of this website. If you find any additional problems not listed above or think we're not meeting accessibility requirements, contact: <a href="mailTo:{{ CONTACT_US_EMAIL_ADDRESS }}">{{ CONTACT_US_EMAIL_ADDRESS }}</a>.</p>
```
Comment (abhi-markan) [inline trade-finance-manager-ui/templates/accessibility-statement.njk:45]: Just to confirm this a tag does not require any formatting class?

**Review by abhi-markan — CHANGES_REQUESTED** [review]

File: docker-compose.gha.yml (hunk 1 above)
Comment (abhi-markan) [reply docker-compose.gha.yml:86]: Please ignore have noticed portal PR

File: trade-finance-manager-ui/templates/accessibility-statement.njk (hunk 2 above)
Comment (AlexBramhill) [reply trade-finance-manager-ui/templates/accessibility-statement.njk:45]: I wouldnt say so, it follows existing formatting for this page in terms of links. Good spot though.

**Review by abhi-markan — APPROVED** [review]

**Comment by AlexBramhill** [comment]
closing and reopening as GHAs are having a moment

**Pull request closed without merging**